grandpa-client/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "grandpa-client"
```

Comment:
prefer if we rename the folder to be just grandpa. We should also export the prover in this repo.